install_bootstrap

```diff
@@ -249,7 +249,15 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
 #            read -ep $'What is the email address for notification? \n' email_address
 #      fi
 
-      
+      project_name='project_name'
+      ask_question "What is the name of the project for this instance (e.g. Stage, Commons, AnVIL, etc.)? This name will be used for error messages." "$PROJECT_NAME" "project name" $project_name
+
+      contact_email='contact_email'
+      ask_question "What is a contact email in case there are problems authenticating?" "$CONTACT_EMAIL" "contact email" $contact_email
+
+      email_whitelist_name='email_whitelist_name'
+      ask_question "What is the name of the AWS Secret that contains the whitelist? (e.g. commons/dev/whitelist)" "$EMAIL_WHITELIST_NAME" "email whitelist name" $email_whitelist_name
```

Comment:
We want to use this as a flag to indicate whether authentication is required to access the instance. The prompt should reflect that, e.g., something like like "leave blank if you do not want to enforce authorization".

Reply:
This is also fixed now.